packages/kit/test/apps/hooks/src/hooks.js

```diff
@@ -0,0 +1,39 @@
+// The following line does not work with vite resolution. Why?
+// import { sequence } from '@sveltejs/kit/hooks';
+import { sequence } from '../../../../dist/hooks';
+
+/** @type {import('@sveltejs/kit').Handle} */
+export const handle = sequence(
+	({ request, next }) => {
+		request.locals.first = true;
+		return next();
```

Comment:
```suggestion
	({ request, resolve }) => {
		request.locals.first = true;
		return resolve(request);
```

Reply:
Deleted the commit which added the `next` function.